### PR TITLE
Use setuptools_scm

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
 graft tests
 
 global-exclude *.py[co] .DS_Store
+prune tests
```
Comment (haleepfl) [inline MANIFEST.in]: Probably don't need to graft (above) and then prune them. If we usually include the tests, then let's just leave them in

File: morph_tool/__init__.py [hunk 2]
```diff
@@ -5,7 +5,6 @@
 from morph_tool.converter import convert
 from morph_tool.morphio_diff import diff
 from morph_tool.spatial import point_to_section_segment
-from morph_tool.version import __version__
```
Comment (haleepfl) [inline morph_tool/__init__.py]: `__version__` is also used in doc/source/conf.py, I think there is a snippet from setuptools_scm specifically for sphinx.

File: MANIFEST.in (hunk 1 above)
Comment (wizmer) [reply MANIFEST.in]: ahahah. So dump. Indeed !

File: morph_tool/__init__.py (hunk 2 above)
Comment (wizmer) [reply morph_tool/__init__.py:8]: it now uses
```python
 version = get_distribution('morph_tool').version
```

**Pull request merged**